network-allocation.md

```diff
@@ -5,16 +5,40 @@ This document summarizes Kubecost network cost allocation, how to enable it, and
 
 When this feature is enabled, Kubecost gathers network traffic metrics in combination with provider-specific network costs to provide insight on network data sources as well as the aggregate costs of transfers.
 
+Metrics include egress and ingress data transfers by pod and are classified as internet, cross-region and cross-zone.
+
 ## Enabling network costs
 
 To enable this feature, set the following parameter in _values.yaml_ during [Helm installation](http://kubecost.com/install):
  ```
  networkCosts.enabled=true
  ```
- You can view a list of common config options [here](https://github.com/kubecost/cost-analyzer-helm-chart/blob/ab384e2eb027e74b2c3e61a7e1733ffa1718170e/cost-analyzer/values.yaml#L276). If you are integrating with an existing Prometheus, you can set `networkCosts.prometheusScrape=true` and the network costs service should be auto-discovered.
- 
+ You can view a list of common config options [here](https://github.com/kubecost/cost-analyzer-helm-chart/blob/700cfa306c8e78bc9a1039b584769b9a0e0757d0/cost-analyzer/values.yaml#L573).
+
+ If using the included Prometheus instance, the scrape is automatically conifgured.
+
+ If you are integrating with an existing Prometheus, you can set `networkCosts.prometheusScrape=true` and the network costs service should be auto-discovered.
+
+ Alternatively a serviceMonitor is also [available](https://github.com/kubecost/cost-analyzer-helm-chart/blob/700cfa306c8e78bc9a1039b584769b9a0e0757d0/cost-analyzer/values.yaml#L716).
+
 > **Note**: Network cost, which is disabled by default, needs to be run as a privileged pod to access the relevant networking kernel module on the host machine.
- 
+
+## Resource limiting
+
+In order to reduce resource usage, Kubecost recommends setting a CPU limit on the network-costs daemonset. This will cause a few seconds delay during peak usage and does not effect overall accuracy. This is done by default in Kubecost 1.99+.
+
+For existing deployments, these are the recommended values:
+
+```
+    resources:
+      limits:
+        cpu: 500m
+      requests:
+        cpu: 50m
+        memory: 20Mi
+```
+ > This write-up regarding CPU limiting <https://home.robusta.dev/blog/stop-using-cpu-limits> has points worth considering. In the case of low-priority/latency insensitive jobs, there is little risk to setting CPU limits.
```

Comment:
any changes that you'd recommend?

Reply:
I removed the note.

I think it causes more confusion than clarity. 

I'm open to adding more color if we think it is needed.